src/components/Chatbot/Chatbot.tsx

```diff
@@ -23,90 +25,180 @@ const NewConversationButton = styled(PrimaryButton)(() => ({
 }));
 
 export const Chatbot = () => {
-  const [conversation, setConversation] = useState<Message[]>([]);
+  const [messages, setMessages] = useState<MessageBubbleProps[]>([]);
   const [error, setError] = useState(false);
   const [loading, setLoading] = useState(false);
+  const [responding, setResponding] = useState(false);
+  const [conversationCreated, setConversationCreated] = useState<string | null>(
+    null
+  );
+  const [errorMessage, setErrorMessage] = useState('Unknown error');
 
-  const addMessageToConversation = async (message: Message) => {
-    setConversation([...conversation, message]);
+  const loadConversationFromBackend = async () => {
+    setLoading(true);
     try {
       setLoading(true);
```

Comment:
Why is `setLoading(true);` called twice?

Reply:
Because i am trying to get it to be set to true but it doesnt work